filters/event_system.go

```diff
@@ -224,13 +219,7 @@ Logs:
 			continue Logs
 		}
 		for i, sub := range topics {
-			match := len(sub) == 0 // empty rule set == wildcard
-			for _, topic := range sub {
-				if log.Topics[i] == topic {
-					match = true
-					break
-				}
-			}
+			match := slices.Contains(sub, log.Topics[i])
```

Comment:
if `sub` is empty, `match` should also be `true`. So i think this should be:

```suggestion
if  len(sub) == 0 || slices.Contains(sub, log.Topics[i]) {
  continue Logs
}
```

Reply:
Modified. Thanks for your review.